gallery/serviceshub/Assessments.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
the existance of this file in a new folder "serviceshub" implies that there's a workbook type "serviceshub" but there isn't.  creation of new workbook types requires workbooks team to do work to create those types, is this file expected to be in the "workbooks" folder instead?

Reply:
Worked on and made changes to code - as per our discussion in the email.